lib/Documentable/CLI.pm6

```diff
@@ -120,10 +121,14 @@ package Documentable::CLI {
         Bool :a(:$all)             = False                    #= Equivalent to -t -p -s -i --search-index
     ) {
         my $beginning = now; # to measure total time
-        if (!"./html".IO.e || !"./assets".IO.e || !"./templates".IO.e and $v) {
-            note q:to/END/;
-                (warning) html and/or assets and/or templates directories
-                cannot be found. You can get the defaults by executing:
+	my $asset-dir-missing = @default-asset-dirs.map({!.IO.e}).any;
+        if ($asset-dir-missing and $v) {
+            note qq:to/END/;
```

Comment:
The original `if` didn't contain `highlights`. This flag does. Is this intentional?

Reply:
Yes, this is intentional, as this is what I understood from your comment earlier today (https://github.com/Raku/Documentable/pull/124#issuecomment-665602848), i.e. that the `highlights` directory probably should be checked for existence as well as the others.